code/ui/Hud.cs

```diff
@@ -51,55 +48,60 @@ private void OnPlayerSpawned(TTTPlayer player)
                 return;
             }
 
-            Current.AliveHudPanel.Enabled = true;
+            AliveHudPanel.Enabled = true;
+            GeneralHudPanel.GetCrosshair().Enabled = true;
         }
 
         [Event("tttreborn.player.died")]
-        private void OnPlayerDied(TTTPlayer player)
+        private void OnPlayerDied(TTTPlayer deadPlayer)
         {
-            if (player != Local.Client.Pawn)
+            if (deadPlayer != Local.Client.Pawn)
             {
                 return;
             }
 
-            Current.AliveHudPanel.Enabled = false;
+            AliveHudPanel.Enabled = false;
+            GeneralHudPanel.GetCrosshair().Enabled = false;
         }
 
         public class GeneralHud : Panel
         {
-            public GeneralHud(Sandbox.UI.Panel parent) : base(parent)
+            public GeneralHud() : base()
             {
-                Parent = parent;
-
-                Parent.AddChild<RadarDisplay>();
-                Parent.AddChild<Crosshair>();
-                Parent.AddChild<PlayerRoleDisplay>();
-                Parent.AddChild<PlayerInfoDisplay>();
-                Parent.AddChild<InventoryWrapper>();
-                Parent.AddChild<ChatBox>();
-
-                Parent.AddChild<VoiceChatDisplay>();
-                Parent.AddChild<GameTimerDisplay>();
-
-                Parent.AddChild<VoiceList>();
+                AddClass("fullscreen");
+                AddChild<Crosshair>();
```

Comment:
Let's throw the crosshair inside of `AliveHud`, since we already disable / enable it based on being alive.

Reply:
Yes, but I tried this and it fails to draw the crosshair on spawn until I toggle the console on and off.

Is there a sync, or refresh I need to perform?

Reply:
Even if I switched to doing one of the following it still fails to draw the crosshair:
```csharp
AliveHudPanel.Enabled = value;
AliveHudPanel.GetCrosshair().Enabled = value;
```

```csharp
AliveHudPanel.Enabled = value;
```

Reply:
Hmm, when you open and close the dev console, the cross-hair appears? I've never ran into something like this. It should work like all of the other HUD elements in `AliveHud`.

Reply:
**After spawning in:**
![image](https://user-images.githubusercontent.com/24533882/133911300-41b399ab-fdc2-4559-b620-9ffb6d25fbbe.png)

**After toggling console on and off:**
![image](https://user-images.githubusercontent.com/24533882/133911307-0857901b-5cc5-4ea2-92d2-4fddecd90dc0.png)

Reply:
I do believe that this is a result of toggling AliveHud on and off, but don't know how I would get around this.

Reply:
I'll hopefully take a look at this sometime and see if I can figure it out.